lib/tests/2wp.js

```diff
@@ -67,7 +67,7 @@ const execute = (description) => {
     after(async () => {
 
         // Ensure there's no pegout txs left to be processed
-        await triggerRelease(rskTxHelpers, btcTxHelper);
+        await triggerRelease(rskTxHelpers, btcTxHelper, {}, false);
```

Comment:
When there's no `release_request_received`, then there's no reason to mine extra blocks for "next pegout creation block number", so passing a `false` flag to avoid mining extra blocks in this case.